frontend/src/core/comments/components/AddComment.js

```diff
@@ -67,8 +84,9 @@ export default function AddComments(props: Props) {
                     name='comment'
                     dir='auto'
                     placeholder={ `Write a comment…` }
+                    rows={ rows }
                     ref={ commentInput }
-                    onKeyDown={ onEnterSubmit }
+                    onKeyUp={ handleKeyUp }
```

Comment:
If you paste text instead of typing it, textarea doesn't resize until you release the last key.

I think using `onChange` instead of `onKeyUp` would fix that.